File: java/control_flow/if_statement/IfStatement.java
class IfStatement {
  public static void main(String args[]) {
    if (1 > 2)
      System.out.print("1 > 2\n");

    if (2 > 1)
      System.out.print("2 > 1\n");

    System.out.print("\n");

    boolean condition;

    condition = false;

    if (condition)
      System.out.print("false: ok\n");

    condition = true;

    if (condition)
      System.out.print("true: ok\n");

    System.out.print("\n");
  }
}

File: pascal/control_flow/if_statement/if_statement.pas
program IfStatement;

var
  condition: boolean;

begin
  if (1 > 2) then
    writeln('1 > 2');

  if (2 > 1) then
    writeln('2 > 1');

  writeln();

  condition := false;

  if (condition) then
    writeln('false: ok');

  condition := true;

  if (condition) then
    writeln('true: ok');

  writeln();
end.
